types/public/package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
A nested package.json, my plan here is to publish everything committed under `types/public`